### PR TITLE
Record history HF5 from existing HF5 files

### PR DESCRIPTION
**Pull request by sophietheis**

### CODE REVIEW

**Comment by glyg** [comment]
cool,

Can you put the new argument at the end in the signature and add the doc? I don't see the use case very clearly? Why do you need to change the `sheet` attribute every time you record?

Also, I don't think the copy is necessary if we don't write the object

**Comment by sophietheis** [comment]
It is used to convert old simulations files system to the new one. 
Sheet argument has to be used only for simulation which already run and to create one hf5 file of simulation instead of x hf5 files.

**Comment by glyg** [comment]
ok I understand now! Maybe it can also prove useful in other situations...

File: tyssue/core/history.py [hunk 1]
```diff
@@ -299,18 +299,23 @@ def time_stamps(self):
...
-    def record(self, to_record=None, time_stamp=None):
+    def record(self, to_record=None, time_stamp=None, sheet=None):
         """Appends a copy of the sheet datasets to the history HDF file.
 
         Parameters
         ----------
         to_report : Deprecated - list of strings
             the datasets from self.sheet to be saved
+        sheet: a :class:`Sheet` object which we want to record. This argument is used
+        to convert several HF5 files of one simulation to only one HF5 files.
```
Comment (glyg) [inline tyssue/core/history.py]: I'd rather write (maybe more generally):
this argument can be used if the sheet object is different at each time point

Comment (sophietheis) [reply tyssue/core/history.py]: I'll change.

**Comment by glyg** [comment]
you can merge if you wabnt

**Pull request merged**